training/finetune_acft_hf_nlp/environments/acpt/context/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#PTCA image
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:{{latest-image-tag}}
+
+# ls root dir
+RUN ls -lh
+
+USER root
+RUN apt-get -y update
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt --no-cache-dir
+RUN pip install azureml-evaluate-mlflow==0.0.1
+RUN python -m nltk.downloader punkt
+
+# Check versions
```

Comment:
nit: do you think you can remove it? dependencies are in the log, and you can always pull an image

Reply:
`pip list` is in a lot of other environments. I'm with you about removing it, though.

Reply:
Some packages are installed by multiple packages and having a pip list in the end has proven useful while testing. That's the reason for keeping it.

Reply:
If this is not a hard requirement, could we take this up in a future PR? Also, the 'azureml-evaluate-mlflow' team is working on upgrading their transformers dependency. Once that is done, we can move this package to requirements.txt file